README.md

```diff
@@ -65,16 +66,32 @@ In order to successfully build and install the plugin, you need to have **ffmpeg
 
 ### RTMP receiver
 
-Server-side example, in which Membrane will act as an RTMP server and receive the stream, can be found under [`examples/source.exs`](examples/source.exs). Run it with:
+Server-side example, in which Membrane element will act as an RTMP server and receive the stream, can be found under [`examples/source.exs`](examples/source.exs). Run it with:
```

Comment:
let's mention that it only allows a single connection

Reply:
done